test/services/dumpers/agent-nodejs/agent-nodejs.unit.test.ts

```diff
@@ -0,0 +1,615 @@
+import type { ConfigInterface } from '../../../../src/interfaces/project-create-interface';
```

Comment:
[SKIP] this file lacks a lot of unit tests a specific configuration (`MySQL`, `MSSQL` for the index file, host for docker compose)

Knowing we have integration tests, and not that much time, let's move on. But in the feature, any business logic should be tested in unit tests AND integration tests as-well